### PR TITLE
CI: update autotyping

### PR DESCRIPTION
**Pull request by twoertwein**
Added many of the `--guess-common-names` guestimations. This new option would still make changes, so I disabled it.

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -4306,7 +4306,7 @@ def __delitem__(self, key) -> None:
     # Unsorted
 
     @final
-    def _check_inplace_and_allows_duplicate_labels(self, inplace):
+    def _check_inplace_and_allows_duplicate_labels(self, inplace: bool_t):
```
Comment (MarcoGorelli) [inline pandas/core/generic.py:4309]: nice 😄

**Review by MarcoGorelli — APPROVED** [review]
nice, thanks @twoertwein !

File: pandas/core/generic.py (hunk 1 above)
Comment (twoertwein) [reply pandas/core/generic.py:4309]: It was `bool`, had to change it to `bool_t`

Comment (MarcoGorelli) [reply pandas/core/generic.py:4309]: yeah that's what I meant, I presume the other hook caught this one?

Comment (twoertwein) [reply pandas/core/generic.py:4309]: mypy made me change it :)

**Pull request merged**